build.gradle.kts

```diff
@@ -16,9 +16,9 @@ buildscript {
         }
     }
 
+    // TODO: Move to plugins block (recommended from AGP 7.3)
     dependencies {
         classpath(Plugins.agp)
```

Comment:
newer version of gradle relies mostly on plugin mechanism, goo to move other as well later but for now only hilt is required